### PR TITLE
update msquic to 2.1.1 for Ubuntu arm64

### PR DESCRIPTION
**Pull request by wfurt**

### CODE REVIEW

**Comment by mthalman** [comment]
@wfurt - Please rebase your branch from main. You're missing some important commits in your PR branch.

**Comment by wfurt** [comment]
updated.

File: src/ubuntu/18.04/helix/arm64v8/Dockerfile [hunk 1]
```diff
@@ -50,19 +50,15 @@ RUN ln -sf /usr/bin/python3 /usr/bin/python && \
...
-    cd /tmp && \
-    rm -r pwsh msquic
+# Add MsQuic
+RUN curl -LO https://packages.microsoft.com/keys/microsoft.asc && \
+    echo 2cfd20a306b2fa5e25522d78f2ef50a1f429d35fd30bd983e2ebffc2b80944fa microsoft.asc| sha256sum --check - && \
+    apt-key add microsoft.asc && \
+    rm microsoft.asc && \
+    apt-add-repository https://packages.microsoft.com/ubuntu/18.04/prod && \
+    apt-get update && \
+    apt-get install -y libmsquic && \
```
Comment (mthalman) [inline src/ubuntu/18.04/helix/arm64v8/Dockerfile:60]: Build is getting an error, not finding this package.

Comment (wfurt) [reply src/ubuntu/18.04/helix/arm64v8/Dockerfile:60]: I will check. The 2.1.1 (and earlier) seems to be there... https://packages.microsoft.com/ubuntu/18.04/prod/pool/main/libm/libmsquic/
(and I can follow-up with msquic)

Comment (wfurt) [reply src/ubuntu/18.04/helix/arm64v8/Dockerfile:60]: oh. arm. it its not there.... ;(

Comment (wfurt) [reply src/ubuntu/18.04/helix/arm64v8/Dockerfile:63]: reported to msquic and I put in workaround with comment for now.

**Review by rzikm — APPROVED** [review]
LGTM!

**Review by mthalman — APPROVED** [review]

**Pull request merged**